runtime/doc/builtin.txt

```diff
@@ -2007,6 +2008,49 @@ complete_info([{what}])				*complete_info()*
 <
 		Return type: dict<any>
 
+complete_match([{lnum}, {col}])			*complete_match()*
+		Returns a List of matches found according to the 'isexpand'
+		option. Each match is represented as a List containing
+		[startcol, trigger_char] where:
+		- startcol: column position where completion should start,
+		  or -1 if no trigger position is found. For multi-character
+		  triggers, returns the column of the first character.
+		- trigger_char: the matching trigger string from 'isexpand',
```

Comment:
The name `trigger_char` may be a bit misleading, as it can be multiple chars.

Reply:
rename to `trigger_text`